boards/esp32-wrover-kit/Kconfig

```diff
@@ -20,6 +20,7 @@ config BOARD_ESP32_WROVER_KIT
     select HAS_PERIPH_PWM
     select HAS_PERIPH_SPI
     select HAS_SDCARD_SPI
-    select MODULE_ESP_JTAG
+
```

Comment:
Please add
```
    select HAS_ESP_JTAG
```
to the list otherwise dependencies fail:
```
[genconfig.py]:ERROR-=> The module MODULE_ESP_JTAG could not be set to y.
[genconfig.py]:ERROR-   Check the following unmet dependencies: HAS_ESP_JTAG (=n)
```

Reply:
The feature is missing on both makefile and Kconfig, but it is not failing on the Makefile-based dependency resolution because the feature is being required on the wrong file
https://github.com/RIOT-OS/RIOT/blob/44ebc38caca3ce24914a6b543fb34703a304e660/cpu/esp32/Makefile.include#L53-L55

I'll move it to the `Makefile.dep` and provide the feature on both Kconfig and Makefile.

Reply:
JTAG dependency fixed in Makefile here bff8cc88b174e4201ff56ce9ca006842ab35fadc. Now the `esp32-wrover-kit` provides the feature (013490280b7c9233630529fe3270f03e6a4a16c1). And for the Kconfig counterpart: 9d0e0d94016e0bed4ab14f492a8df2f05d9d6d5e